ykrt/src/mt.rs

```diff
@@ -219,215 +217,206 @@ impl MTThread {
         &mut self,
         loc: &Location<I>,
     ) -> Option<fn(&mut I) -> *mut RawStopgapInterpreter> {
-        // Since we don't hold an explicit lock, updating a Location is tricky: we might read a
```

Comment:
A few lines above we have `statemachine` which should be `state machine`.

Reply:
Huh? I'm not sure what this is talking about. That said, the comment here is clearly incorrect, and the Ordering is unnecessarily strong. Fixed in 80db78f86a108897087f83b958a6ede52671685e.

Reply:
Line 215: `/// `Location`s represent a statemachine`

Reply:
Um, and ...? :)